crates/aptos-crypto/benches/bulletproofs_batch_vrfy.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use]
+extern crate criterion;
+
+use aptos_crypto::bulletproofs::MAX_RANGE_BITS;
+use bulletproofs::{BulletproofGens, PedersenGens, RangeProof};
+use criterion::{measurement::Measurement, BenchmarkGroup, BenchmarkId, Criterion, Throughput};
+use curve25519_dalek_ng::scalar::Scalar;
+use merlin::Transcript;
+use rand::{thread_rng, Rng};
+
+fn get_values(num_bits: usize, batch_size: usize) -> (Vec<u64>, Vec<Scalar>) {
+    let mut rng = thread_rng();
+
+    let v = (0..batch_size)
+        .map(|_| rng.gen_range(0u64, (2u128.pow(num_bits as u32) - 1u128) as u64))
+        .collect::<Vec<u64>>();
+
+    // Sigh, some RngCore incompatibilites I don't want to deal with right now.
+    let b = (0..batch_size)
+        .map(|_| Scalar::hash_from_bytes::<sha3::Sha3_512>(b"some random blinder"))
```

Comment:
Don't you want **different** scalars here?

Reply:
Done